front/lib/api/assistant/configuration.ts

```diff
@@ -219,6 +224,27 @@ async function fetchAgentConfigurationsForView(
         ...baseAgentsSequelizeQuery,
         where: baseWhereConditions,
       });
+    case "archived":
+      // Get the latest version of all archived agents.
+      // For each sId, we want to fetch the one with the highest version, only if it's status is "archived".
+      return AgentConfiguration.findAll({
```

Comment:
`nit` but we could have done on single SQL query.

Reply:
How :) ? It's def not that easy, unless you do a big raw SQL query with a subquery.